docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,67 @@
+# Capture
+
+
+Capture snapshot the current DALi scene. The captured result can be saved as a file and also can be used to be rendered by DALi `ImageView` or `ImageVisual`.
```

Comment:
Suggest to rephrase: Is this correct?
The `Dali::Capture` is a control to capture a snapshot of the current scene. The captured result can be saved as a file and rendered by DALi `ImageView` or `ImageVisual`.

Reply:
Thank you good point, but Dali::Capture is not a control that the word used in the DALi.
So to avoid of confusing, I modified it as follow

The Dali::Capture is a class to capture a snapshot of the current scene. The captured result can be saved as a file and rendered by DALi `ImageView` or `ImageVisual`.